examples/C++/espresso.cpp

```diff
@@ -59,19 +59,19 @@ int main() {
 	std::thread sub_thread(subscriber_thread, std::ref(context));
 
 	// Set up listener thread
-	std::thread listener_thread(listener_thread, std::ref(context));
+	std::thread lis_thread(listener_thread, std::ref(context));
 
 	// Main thread acts as the listener proxy
 	zmq::socket_t proxy(context, ZMQ_PAIR);
 	proxy.bind("inproc://listener");
 
 	// Proxy messages between SUB and PUB sockets
-	zmq_proxy(zmq::socket_t(context, ZMQ_XSUB), zmq::socket_t(context, ZMQ_XPUB), proxy);
+//	zmq_proxy(zmq::socket_t(context, ZMQ_XSUB), zmq::socket_t(context, ZMQ_XPUB), proxy);

```

Comment:
Can any one please help me with this? This statement looks like a no-op. The behavior is same before and after this change. Also I am not sure how capturing works with zmq_proxy in C++. there is no valid example that has capture socket as non null in C++. Please help me with an example here.